docs/nlu/components.rst

```diff
@@ -621,7 +621,7 @@ LexicalSyntacticFeaturizer
     .. code-block:: yaml
 
         pipeline:
-        - name: "LexicalSyntacticFeaturizer":
+        - name: LexicalSyntacticFeaturizer
           "features": [
```

Comment:
```suggestion
          features: [
```
This still works right? I think it looks more like a second dictionary key that way, which is easier to understand